arch/arm/soc/nordic_nrf/nrf52/soc.c

```diff
@@ -26,6 +26,8 @@ extern void _NmiInit(void);
 #endif
 
 #if defined(CONFIG_SOC_NRF52832)
+#include <system_nrf52810.h>
```

Comment:
@anangl take a look here: I simply added nRF52810 nrf header, in soc.c.
Is there anything else, (besides defining DEVELOP_IN_NRF52832) that we need to do here?

Reply:
Good. But the line above should be `#if defined(CONFIG_SOC_NRF52810)`.
And that should be it.

Reply:
ups, my bad, thanks for catching this.
Will fix.